server/neptune/workflows/internal/deploy/revision/revision.go

```diff
@@ -136,23 +141,22 @@ func (n *Receiver) createCheckRun(ctx workflow.Context, id, revision string, roo
 		summary = fmt.Sprintf("This deploy is locked from a manual deployment for revision %s.  Unlock to proceed.", lock.Revision)
 	}
 
-	var resp activities.CreateCheckRunResponse
-	err := workflow.ExecuteActivity(ctx, n.activities.GithubCreateCheckRun, activities.CreateCheckRunRequest{
+	cid, err := n.checkRunClient.CreateOrUpdate(ctx, id, notifier.GithubCheckRunRequest{
```

Comment:
No versioning here?

Reply:
we shouldn't have to, it's always going to create a check run either way, the activity will be the same.